example_remoteVerify_test.go

```diff
@@ -0,0 +1,77 @@
+package notation_test
+
+import (
+	"context"
+	"fmt"
+
+	_ "github.com/notaryproject/notation-core-go/signature/cose"
+	_ "github.com/notaryproject/notation-core-go/signature/jws"
+	"github.com/notaryproject/notation-go"
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/internal/mock"
+	"github.com/notaryproject/notation-go/verifier"
+	"github.com/notaryproject/notation-go/verifier/trustpolicy"
+	"github.com/notaryproject/notation-go/verifier/truststore"
+)
+
+var (
+	// examplePolicyStatement is an example of a valid trust policy statement.
+	examplePolicyStatement = trustpolicy.TrustPolicy{
+		Name:                  "test-statement-name",
+		RegistryScopes:        []string{"registry.acme-rockets.io/software/net-monitor"},
+		SignatureVerification: trustpolicy.SignatureVerification{VerificationLevel: "strict"},
+		TrustStores:           []string{"ca:valid-trust-store", "signingAuthority:valid-trust-store"},
+		TrustedIdentities:     []string{"x509.subject:CN=Notation Test Root,O=Notary,L=Seattle,ST=WA,C=US"},
+	}
+
+	// examplePolicyDocument is an example of a valid trust policy document.
+	examplePolicyDocument = trustpolicy.Document{
+		Version:       "1.0",
+		TrustPolicies: []trustpolicy.TrustPolicy{examplePolicyStatement},
+	}
+
+	// exampleRemoteVerifyOptions is an example of notation.RemoteVerifyOptions.
+	exampleRemoteVerifyOptions = notation.RemoteVerifyOptions{
+		ArtifactReference:    exampleArtifactReference,
+		MaxSignatureAttempts: 50,
+	}
+)
+
+// Example_remoteVerify demonstrates how to use notation.Verify to verify
```

Comment:
```suggestion
// ExampleRemoteVerify demonstrates how to use notation.Verify to verify
```
example: https://go.dev/src/sort/example_keys_test.go